utils/config.go

```diff
@@ -28,6 +28,7 @@ var (
 			DisableNetworkConfiguration:   Tribool(0),
 			DisableSystemConfiguration:    Tribool(0),
 			ViamServerStartTimeoutMinutes: Timeout(time.Minute * 10),
+			ViamServerExtraEnvVars:        nil,
```

Comment:
Please make sure to add this (and in the future, any other new config fields) to examples/agent-config.jsonc